Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -291,6 +281,23 @@ private extension MoyaProvider {
     }
 }
 
+private extension Array where Element == MultipartFormData {
```

Comment:
Maybe [MultipartFormData.swift](https://github.com/Moya/Moya/blob/master/Sources/Moya/MultipartFormData.swift) would be better for this extension?